modules/tensor_mechanics/src/bcs/PenaltyInclinedNoDisplacementBC.C

```diff
@@ -42,23 +48,38 @@ PenaltyInclinedNoDisplacementBC::computeQpResidual()
   for (unsigned int i = 0; i < _ndisp; ++i)
     v += (*_disp[i])[_qp] * _normals[_qp](i);
 
-  return _penalty * _test[_i][_qp] * v * _normals[_qp](_component);
+  auto penalty = _penalty;
+
+  if (_normalize_penalty)
+    penalty /= _current_side_volume;
```

Comment:
Per our conversation, maybe we need to reverse true/false to match the contact behavior, and do some additional testing.